apps/meteor/client/views/admin/users/EditUser.js

```diff
@@ -131,10 +131,10 @@ function EditUser({ data, roles, onReload, ...props }) {
 				<Field.Row>
 					<Box display='flex' flexDirection='row' justifyContent='space-between' w='full'>
 						<Margins inlineEnd='x4'>
-							<Button flexGrow={1} type='reset' disabled={!canSaveOrReset} onClick={reset}>
+							<Button danger flexGrow={1} type='reset' disabled={!canSaveOrReset} onClick={reset}>
```

Comment:
Let's avoid the usage of danger variation for reset action

Reply:
Got it, I'll fix it, thanks